pom.xml

```
@@ -71,7 +71,7 @@
     <dependency>
       <groupId>uk.gov.ons.ssdc</groupId>
       <artifactId>ssdc-shared-sample-validation</artifactId>
-      <version>1.6.0</version>
+      <version>1.6.1-SNAPSHOT</version>
```

We need to use the release versions of the validation so you can remove the SNAPSHOT. It get's built when the sample-validation PR gets merged
```suggestion
      <version>1.6.1</version>
```